src/Notecard.cpp

```diff
@@ -354,6 +354,23 @@ bool Notecard::sendRequest(J *req)
     return NoteRequest(req);
 }
 
+/**************************************************************************/
+/*!
+    @brief  Sends a request to the Notecard, retrying it on failure until the
+            provided timeout interval lapses.
+    @param    req
+              A `J` JSON request object.
+    @param    timeoutSeconds
+              The timeout interval, in seconds.
+    @return `True` if the message was successfully sent to the Notecard,
+            `False` if the message couldn't be sent.
+*/
+/**************************************************************************/
+bool Notecard::sendRequestWithRetry(J *req, uint32_t timeoutSeconds)
+{
+    return NoteRequestWithRetry(req, timeoutSeconds);
```

Comment:
I assume there is a typo in `note-c`, but when I did a quick check the parameter descriptions don't line up. `NoteRequestWithRetry()` indicates it is using **milliseconds** instead of seconds.

```c
bool NoteRequestWithRetry(J *req, uint32_t timeoutms);
```

https://github.com/blues/note-c/blob/master/note.h#L133

Reply:
Yeah that typo has been in the header since inception. It's seconds. Will do a note-c PR.